src/client/context-store/board-store-context.js

```diff
@@ -1,36 +1,216 @@
 import React, {useState} from 'react';
+import boardController from '../client-board-controller.js';
 
 export const BoardStoreContext = React.createContext(null);
 
 export default ({children}) => {
     const [board, setBoard] = useState(null);
 
-    const addColumn = (columnName) => {
+    const addColumn = async (columnName) => {
+        const newColumn = await boardController.addColumn(columnName);
+
+        if (newColumn) {
```

Comment:
It seems really strange for me that in UI we should do the same changes with data that are done in server part.
But I don't have really good solutions how to handle it 😄 
@alex-zh WDYT?

Reply:
Agree. I don't like it either. My question is also what is the best approach to react on server responses. What if it returns an error how better to rollback the changes? @alex-zh?